docs/src/routes/docs/[...1]overview/[...2]onboard.js-migration-guide.md

```diff
@@ -89,7 +89,7 @@ We've also added more expansive custom CSS properties so you can style every par
 }
 ```
 
-The full list of CSS variables for web3-onboard are available here.
+The full list of CSS variables for web3-onboard are available [here](https://onboard.blocknative.com/docs/packages/core#custom-styling).
```

Comment:
You could add a link to our custom styling tool - https://onboard.blocknative.com/theming-tool
Its shown as an experimental option [here](https://onboard.blocknative.com/docs/getting-started/customization) as an example not that it needs to be copied